NAS2D/Renderer/Renderer.cpp

```diff
@@ -69,23 +55,12 @@ void Renderer::drawTextShadow(const Font& font, std::string_view text, Point<flo
 }
 
 
-/**
- * Sets the color of the fade.
- *
- * \param	color	A Color.
- */
 void Renderer::fadeColor(Color color)
 {
 	mFadeColor = color;
 }
 
 
-/**
- * Non-blocking screen fade.
- *
- * \param	fadeInTime	Time in miliseconds the fade should last. A value of 0
- *					will instantly fade the screen in.
```

Comment:
I'm on the fence about the value of this comment.

Reply:
If you aren't sure whether or not it's a bad comment, it probably is. :P

Reply:
From another perspective, it we're uncertain if something should be changed, it probably shouldn't be changed. :wink:

Reply:
Asking "Do we need this?" or "Is this useful?" and if you can't come up with a good reason immediately, it likely can be cut.

In this case, the comment restated what already could be explicitly stated in the function and argument names. So I delete it.

Reply:
The "non-blocking" part might not be obvious. The treatment of `0` and whether or not it's a valid value might not be obvious.

Reply:
Could you explain what "instantly fade" means? does that mean there is no fade at all or it cuts to the fade color for one frame?

Reply:
This is part of why I want to defer on making edits to this comment. Maybe we should worry about the low hanging fruit first, where we're all in agreement that a comment has no real value and should be removed. That way we could get the PR merged more quickly, and don't have a large set of changes hanging around unmerged, and potentially accumulating merge conflicts.

For more contentious parts we can either discuss in the issue (and leave the issue open), create a new issue to deal with that (and close the old one after the bulk of changes are merged), or maybe even just ignore it if other future work is likely to make the point obsolete. For instance, there is some work to extract the fade ability to a separate control, which may or may not make some of these discussions obsolete.